package.json

```diff
@@ -1,19 +1,19 @@
 {
   "name": "movies-db",
   "version": "1.0.0",
-  "description": "react-redux movies database",
+  "description": "react movie database",
   "main": "index.js",
   "scripts": {
     "start": "webpack-dev-server",
-    "build:dev": "webpack --env dev",
-    "build:prod": "webpack --env prod",
+    "build:dev": "webpack",
```

Comment:
If mode is not specified it is "production" by default. More adjustments for development process https://webpack.js.org/guides/development/

Reply:
I see you set "mode" inside webpack.config.js, but it still can be not straightforward to other developers :)

Reply:
Did I understand you correctly?